js/rpg_managers/DataManager.js

```diff
@@ -41,6 +41,8 @@ DataManager._globalId       = 'RPGMV';
 DataManager._lastAccessedId = 1;
 DataManager._errorUrl       = null;
 
+DataManager.imageReservationId  = Utils.generateRuntimeId();
```

Comment:
After all I thought that `DataManager` need not have its own ID because this method is called from `Scene_File` 😇